source/common/json/json_internal.cc

```diff
@@ -788,6 +788,10 @@ std::string Factory::serialize(absl::string_view str) {
   return j.dump();
 }
 
+std::vector<uint8_t> Factory::jsonToMsgpack(const std::string& json_string) {
+  return nlohmann::json::to_msgpack(nlohmann::json::parse(json_string, nullptr, false));
```

Comment:
Hah, the `allow_exceptions` is set to false. It's not easy to identify that when doing review. 🤣